### PR TITLE
Blaine/grocery list

### PR DESCRIPTION
**Pull request by BlaineSieck**
Styling changes and grocery list is now scrollable

### CODE REVIEW

**Review by ynigoreyes — APPROVED** [review]

File: src/screens/grocerylist/GroceryList.jsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            console.error(err);
+            logError(err);
+        }
+    }
+
+    async function removeTodo(index) {
+        try {
+            let newList = await groceryListService.removeFoodItemFromContainer(grocList, index);
+            setGrocList(newList);
+            const newTodos = [...todos];
```
Comment (BlaineSieck) [inline src/screens/grocerylist/GroceryList.jsx:82]: @ynigoreyes 

I'm fairly sure the problem is here. For now, I'm using the hamburger icon as a temporary delete button. So when you press that button it deletes the item from the list by passing the index of the item in the todos array to the removeTodo() function. The correct item is always deleted from the backend, but it only appears to work correctly on the front end when you delete the item at the bottom of the list. Basically, on the front end no matter what item you try to delete, it removes the item from the end of the list. I think the setTodos() call may not be working?

I've confirmed that the correct index is being passed so I'm probably just missing something!

Comment (ynigoreyes) [reply src/screens/grocerylist/GroceryList.jsx:82]: Why do we have a todo list and a groc list? Are you trying to keep two separate lists?

File: src/screens/grocerylist/GroceryList.jsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            newTodos.splice(index, 1);
```
Comment (ynigoreyes) [inline src/screens/grocerylist/GroceryList.jsx:83]: This is where I think there may be a problem, but I also just may not understand what's going on. array.splice just replaces/removes at a location in the array. It looks like you are just cutting off the end of each `todo` list. If you are trying to delete an element with a matching condition, I would suggest `Array.prototype.filter()` you can find that [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)

File: src/screens/grocerylist/GroceryList.jsx (hunk 1 above)
Comment (BlaineSieck) [reply src/screens/grocerylist/GroceryList.jsx]: groclist is a different object, the todo list is the array of foodItems within the grocery list

File: src/screens/grocerylist/GroceryList.jsx (hunk 2 above)
Comment (BlaineSieck) [reply src/screens/grocerylist/GroceryList.jsx]: This is fixed now

**Review by ynigoreyes — APPROVED** [review]

**Pull request merged**